api/src/test/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -234,6 +243,17 @@ private K waitFor(String resource, String name, Predicate<JsonNode> ready) {
         return (K) this;
     }
 
+    @Override
+    public K waitForDeployment(String name, int expected) {
```

Comment:
All tests should be now using the new version.

Reply:
Good question, but TBH I think it will be really difficult to manage this in the StrimziRunner. For this, we need to additionally parse `.yamls` files or ... ?